examples/flutter_gallery/lib/gallery/home.dart

```diff
@@ -300,13 +300,6 @@ class _GalleryHomeState extends State<GalleryHome> with SingleTickerProviderStat
     super.dispose();
   }
 
-  static Widget _animatedSwitcherLayoutBuilder(List<Widget> children) {
```

Comment:
I removed this because now that it has a center alignment (it was created originally with a different alignment), it is identical to the default builder, and isn't needed anymore.